src/libs/actions/Session.js

```diff
@@ -234,6 +237,27 @@ function signIn(password, twoFactorAuthCode) {
         });
 }
 
+function signInWithShortLivedToken(accountID, email, shortLivedToken) {
```

Comment:
- Don't we need to call authenticate with the short lived token before creating a login?
- We're calling `signInWithShortLivedToken(shortLivedToken)` and not passing encrypted auth token, that means it'll be set to undefined in `setSuccessfulSignInData` right?